### ISSUE TITLE
Enabling 2FA should include some recovery codes

### ISSUE THREAD

**Issue by cellio** [codidact/qpixel#266]
https://software.codidact.com/questions/277158

> After enabling two factor authentication (2FA) for my account I'm only notified It is successfully turned On.
>
> I think it should include some recovery codes as well (to be used in case I can't access my authenticator app, etc.).

@ArtOfCode- wrote in a comment:

> Just realised I never actually responded to this. It's not one we thought of (the 2FA system was copied over from another, older, implementation of mine from before it was a recommended practice), but it's definitely one we should do. — ‮edoCfOtrA‭ 3 months ago

**Comment by ArtOfCode-** [comment]
Re-prioritised - not critical, manual support is sufficient for now until we have spare time to do this

**Comment by cellio** [comment]
Oh, we have a path for manual support?  In that case I concur.

**Comment by ArtOfCode-** [comment]
Yep - if someone gets locked out, poke me and I can restore access

**Comment by cellio** [comment]
We had a case where we had to invoke manual support and it left us a little uneasy about validation.  I'm raising the priority of this again.  We should do *something* -- even if the process remains partly manual, being able to generate and store a code that we also give to the user would meet the spirit.  We don't have to automate account recovery with a code if "contact support with your code and we'll check" works for now, but I think we need a code.  Or some sort of shared secret.
